docs/index.md

```diff
@@ -0,0 +1,13 @@
+```{toctree}
+---
+caption: Contents of this website
+maxdepth: 2
+hidden: true
+---
+
+network_area_diagrams/index.md
+single_line_diagrams/index.md
+README.md
+```
+
+# powsybl-diagram
```

Comment:
We could add the one line from the repo readme to introduce powsybl-diagram https://github.com/powsybl/powsybl-diagram/blob/main/README.md?plain=1#L9